dev-env-setup.md

```diff
@@ -0,0 +1,59 @@
+# Set Up Your Starport Development Environment 
+
+To ensure you have a successful experience developing with Starport, make sure that your local system meets these technical requirements.
+
+Starport is supported for the following operating systems:
+
+- GNU/Linux
+- macOS
+- Windows Subsystem for Linux (WSL)
+
+## Install Go
+
+This installation method removes existing Go installations, installs Go in `/usr/local/go/bin/go`, and sets the environment variables.
```

Comment:
`/usr/local/go/bin/go` might be different on some systems, can we deduct this claim?

Reply:
another comment from you @ilgooz that was gathering cobwebs, so sorry to miss this for such a long time